app/views/course/assessment/answer/multiple_responses/_multiple_response.html.slim

```diff
@@ -7,6 +7,6 @@ h2 = format_inline_text(question.title)
                           display_solution: can?(:grade, multiple_response.answer),
                           label_method: :option, label: false, disabled: readonly
 
-- if f.object.auto_grading && f.object.auto_grading.result
-  div
-    = format_html(f.object.auto_grading.result['messages'])
+- if f.object.graded?
```

Comment:
Wait, `!attempting` is not quite suitable, because `answer.correct?` only makes sense after auto_grading. Think still need to be `f.object.auto_grading && f.object.auto_grading.result`.